### PR TITLE
[Beam-9063]update documentation

### PR DESCRIPTION
**Pull request by Hannah-Jiang**
**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Spark/lastCompletedBuild/)
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Java11/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Spark_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_SparkStructuredStreaming/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python2/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python35/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python36/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python37/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python2_PVR_Flink_Cron/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python35_VR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Spark/lastCompletedBuild/)
XLang | --- | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_XVR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_XVR_Spark/lastCompletedBuild/)

Pre-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

--- |Java | Python | Go | Website
--- | --- | --- | --- | ---
Non-portable | [Build Status](https://builds.apache.org/job/beam_PreCommit_Java_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_Cron/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PreCommit_PythonLint_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/) 
Portable | --- | [Build Status](https://builds.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/) | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

### CODE REVIEW

**Comment by Hannah-Jiang** [comment]
R: @ibzib

File: website/src/documentation/runtime/environments.md [hunk 1]
```diff
@@ -166,20 +166,18 @@ To change the repository, set the `docker-repository-root` option to a new locat
...
 
-1. Sign in to your Docker registry:
 ```
-docker login
+Upload it to the remote repository:
 ```
-2. Navigate to the local copy of your container image and upload it to the remote repository:
-```
-docker push apache/beam_python2.7_sdk
+docker push $USER-docker-apache.bintray.io/beam/python:beam_python2.7_sdk
```
Comment (ibzib) [inline website/src/documentation/runtime/environments.md]: When I search the project for `bintray`, I only get a few matches:
```
buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy:          configuration.root = "${System.properties["user.name"]}-docker-apache.bintray.io/beam"
website/src/documentation/runtime/environments.md:./gradlew :sdks:python:container:py2:docker -Pdocker-repository-root=$USER-docker-apache.bintray.io/beam/python
website/src/get-started/wordcount-example.md:                      --worker_harness_container_image=apache-docker-beam-snapshots-docker.bintray.io/beam/go:20180515
website/src/get-started/wordcount-example.md:            --worker_harness_container_image=apache-docker-beam-snapshots-docker.bintray.io/beam/go:20180515
```
Of which it looks like the latter two are outdated.

Seeing as Bintray no longer has any special significance, maybe we should change the example repo to Docker hub. e.g. `docker push $YOUR_REPO:beam_python2.7_sdk`

Comment (Hannah-Jiang) [reply website/src/documentation/runtime/environments.md]: Updated it. PTAL when you have time.

File: website/src/documentation/runtime/environments.md [hunk 2]
```diff
@@ -150,34 +151,36 @@ apache/beam_go_sdk                 latest              30cf602e9763        2 wee
...
 ### Overriding default Docker targets
 
-The default [tag](https://docs.docker.com/engine/reference/commandline/tag/) is `latest` and the default repositories are in the Docker Hub `apachebeam` namespace. The `docker` command-line tool implicitly [pushes container images](#pushing-container-images) to this location.
+The default [tag](https://docs.docker.com/engine/reference/commandline/tag/) is sdk_version defined at [gradle.properties](https://github.com/apache/beam/blob/master/gradle.properties) and the default repositories are in the Docker Hub `apache` namespace. 
+The `docker` command-line tool implicitly [pushes container images](#pushing-container-images) to this location.
 
 To tag a local image, set the `docker-tag` option when building the container. The following command tags a Python SDK image with a date.
 ```
-./gradlew :sdks:python:container:py2:docker -Pdocker-tag=2019-10-04
+./gradlew :sdks:python:container:py3.6:docker -Pdocker-tag=2019-10-04
```
Comment (ibzib) [inline website/src/documentation/runtime/environments.md]:
```suggestion
./gradlew :sdks:python:container:py36:docker -Pdocker-tag=2019-10-04
```

File: website/src/documentation/runtime/environments.md [hunk 3]
```diff
@@ -150,34 +151,36 @@ apache/beam_go_sdk                 latest              30cf602e9763        2 wee
...
 ```
 
-To change the repository, set the `docker-repository-root` option to a new location. The following command sets the `docker-repository-root` to a Bintray repository named `apache`.
+To change the repository, set the `docker-repository-root` option to a new location. The following command sets the `docker-repository-root` 
+to a repository named `example-repo` on Docker Hub.
 ```
-./gradlew :sdks:python:container:py2:docker -Pdocker-repository-root=$USER-docker-apache.bintray.io/beam/python
+./gradlew :sdks:python:container:py3.6:docker -Pdocker-repository-root=example-repo
```
Comment (ibzib) [inline website/src/documentation/runtime/environments.md]:
```suggestion
./gradlew :sdks:python:container:py36:docker -Pdocker-repository-root=example-repo
```

File: website/src/documentation/runtime/environments.md [hunk 4]
```diff
@@ -99,6 +99,7 @@ python -m apache_beam.examples.wordcount \
 --environment_config=path/to/container/image
 ```
 
+(Customized containers are not supported with Dataflow runner yet.)
```
Comment (ibzib) [inline website/src/documentation/runtime/environments.md]: If custom containers are not supported yet, do the instructions below this work?

**Comment by ibzib** [comment]
There is also a reference to `apachebeam` that needs to be changed here: https://github.com/apache/beam/pull/10952/files#diff-ba4c631d44bbfe83aaf1b9246b0bf620R41

File: website/src/documentation/runtime/environments.md (hunk 4 above)
Comment (Hannah-Jiang) [reply website/src/documentation/runtime/environments.md]: I think it's still better to include it, it is telling users Dataflow will support custom containers in future, though they are not supported at the moment.

Comment (ibzib) [reply website/src/documentation/runtime/environments.md]: Can we at least make that more clear? It's confusing to document features we don't support yet.

Comment (Hannah-Jiang) [reply website/src/documentation/runtime/environments.md]: I removed dataflow part. Can you please take a look?
(Sorry for long due...)

**Pull request merged**